src/util/Constants.js

```diff
@@ -60,7 +60,8 @@ exports.DefaultOptions = {
   /**
    * WebSocket options (these are left as snake_case to match the API)
    * @typedef {Object} WebsocketOptions
-   * @property {number} [large_threshold=250] Number of members in a guild to be considered large
+   * @property {number} [large_threshold=250] Number of members in a guild where offline users will no longer be sent
```

Comment:
We can structure the description as "A number of members in a guild (offline members are not included), above which guild will be considered large." for more information.

Reply:
But that still leaves initial problem. You mention some treshold after which guild becomes large, but there is still no explanation what in the world `large` does, because the `where offline members are not included` simply says that offline members are not counted towards that treshold

Reply:
Right. "The total number of members in the guild above which the guild will be considered large (between 50 and 250) where offline members are not sent in the guild member list." looks good.

Reply:
IMO what I typed is clearer

Reply:
On the other side it also doesn't really say much. Someone can see `...users will no longer be sent` and will try to set it to something around `9999999999` because they will think you will never be able to get users/members from such guild.

It lacks something like `...no longer be sent initially`, or `on initial guild's loading`, so something that still says what passing the treshold does, but not in a way it could be misinterpreted like in a way i did in comment above  (so not eg. `on first guild's fetch`)

Reply:
Please see @tipakA's previous comments on how to improve the documentation here.

Reply:
Looks this comment only showed the first line of the description, not the full description, which is
> Number of members in a guild where offline users will no longer be sent in the guild member list, between 50 and 250

. I thought it would be good to stick to what Discord says, would
> Number of members in a guild where offline users will no longer be sent in the guild member list when the guild loads, between 50 and 250

be better?

Reply:
@tipakA @amishshah ^ any response?

Reply:
`Number of total members in a guild after which offline users will no longer be sent in the initial guild member list, between 50 and 250` sounds the most accurate to me.. Thoughts @amishshah @tipakA?

Reply:
To propose a different wording: `Number of online users in a guild after which offline ones will no longer be sent in the initial guild member list, between 50 and 250`.

I first thought of "Number of non-offline" but the opposite of that is online, and it was duplicating the word "offline". Thoughts?

Reply:
It isn't online members but total members in a guild. Per discord's docs: `value between 50 and 250, total number of members where the gateway will stop sending offline members in the guild member list`

Reply:
That says something different though? API docs says its the total number of members, not the number of online users (also there could be confusion whether idle/dnd are counted in online, but the discord ui includes those so ig its fine)

Reply:
Ah, what about this? Picking bits from @vladfrangu's and Discord's suggestion:
```suggestion
   * @property {number} [large_threshold=250] Number of members in a guild after which offline users will no longer be sent in the initial guild member list, must be between 50 and 250
```

"Number of total members" sounds weird to me.

Reply:
Sounds good to me. Probably would use `amount` instead of `number`, but i'm aware that this isn't an essay.

Reply:
I'm pretty sure number is correct in this instance
> number is used for things that can be counted, while amount is used for things that cannot be counted

Reply:
> I'm pretty sure number is correct in this instance
> 
> > number is used for things that can be counted, while amount is used for things that cannot be counted

You are right, per https://www.enago.com/academy/understanding-the-difference-between-number-and-amount/

> The answer lies in the English distinction between count and non-count nouns. Although number and amount have similar meanings, number is used for things that can be counted, while amount is used for things that cannot be counted.

I edited the comment to delete the suggestion and let it fallback to the previous, which uses "number".